### DISCUSSION TITLE
novelWriter on Linuxcapable.com

### DISCUSSION THREAD

**Discussion by johnblommers** [vkbo/novelWriter#1093] [General]
novelWriter is becoming better known. Congratulations.

See the very recent article [How to Install novelWriter on Ubuntu 22.04 LTS](https://www.linuxcapable.com/how-to-install-novelwriter-on-ubuntu-22-04-lts/)

\- John

**Comment by vkbo** [comment]
I saw your message, but forgot to respond. Thanks for letting me know!

I've been moving to a new city and new apartment, so I've been rather absent lately. :)
